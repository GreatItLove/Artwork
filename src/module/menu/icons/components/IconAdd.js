import React from 'react';
import Avatar from 'material-ui/Avatar';
import ContentAddCircleOutline from 'material-ui/svg-icons/content/add-circle-outline';
import muiThemeable from 'material-ui/styles/muiThemeable';

/**
 * @function IconAdd
 * @returns {XML}
 */
function IconAdd({ muiTheme }) {
  return (
    <Avatar
      backgroundColor={muiTheme.palette.colorBrandGrey}
      color={muiTheme.palette.colorBrandBlue}
      icon={<ContentAddCircleOutline />}
      size={35}
    />
  );
}

export { IconAdd as IconAddTest };

export default muiThemeable()(IconAdd);
