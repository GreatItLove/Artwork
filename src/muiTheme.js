import getMuiTheme from 'material-ui/styles/getMuiTheme';

const muiTheme = getMuiTheme({
  palette: {
    primary1Color: '#258df2',
    accent1Color: '#40c741',
    colorBrandBlue: '#1e6291',
    colorBrandGrey: '#e7e4d8',
    colorBrandOrange: '#e18927'
  }
});

export default muiTheme;
