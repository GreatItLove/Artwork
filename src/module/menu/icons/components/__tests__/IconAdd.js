import React from 'react';
import { shallow } from 'enzyme';
import toJSON from 'enzyme-to-json';

import { IconAddTest as IconAdd } from '../IconAdd';
import muiTheme from '../../../../../muiTheme';

describe('Icon Add', () => {
  it('Snapshot of IconAdd', () => {
    const wrapper = shallow(<IconAdd muiTheme={muiTheme} />);
    expect(toJSON(wrapper)).toMatchSnapshot();
  });
});
