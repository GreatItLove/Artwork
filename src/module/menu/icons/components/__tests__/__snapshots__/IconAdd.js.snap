// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Icon Add Snapshot of IconAdd 1`] = `
<Avatar
  backgroundColor="#e7e4d8"
  color="#1e6291"
  icon={<ContentAddCircleOutline />}
  size={35}
/>
`;
